### ISSUE TITLE
Improve README

### ISSUE THREAD

**Issue by DominiqueMakowski** [tszanalytics/BayesTesting.jl#6]
Related to #2 and #3, the README could be more attractive with badges exhibiting the package "health", and separate sections for a small description, short examples and links to resources.

**Comment by tszanalytics** [comment]
Yes, I agree.  I have been meaning to look into using Documenter.jl to make a nicer set of docs, but haven't had time yet.  I don't know how to make badges either.
